R/standardize_metadata.R

```diff
@@ -17,5 +17,8 @@ standarize_metadata <- function(metadata) {
     "custom_alignment", "question_number", "matrix_group_name", "matrix_ranking",
     "field_annotation"
   )
+
+  metadata$select_choices_or_calculations[metadata$select_choices_or_calculations == "" &
```

Comment:
I think you should move this functionality into get_categorical_field_labels.R.  I'd like this function to be only about renaming the columns. Consider merging PR #4 first.  If you do you can turn on the yesno tests in test_get_categorical_field_labels.R